mobileapp/src/components/screens/notification/NotificationDetailScreen/NotificationDetailScreen.tsx

```diff
@@ -24,7 +25,24 @@ export const NotificationDetailScreen: React.FC<
   const [processing, setProcessing] = useState(false)
   const [errorDialog, setErrorDialog] = useState(false)
   const { user, logoutUser } = useContext(UserContext)
+  const [details, setDetails] = useState<NotificationDetails>()
 
+  /**
+   * Method to get data from API
+   */
+  const notificationDetails = async () => {
+    const { data } = await api.notifications.getNotificationDetails(
```

Comment:
brakuje obsługi błędów, co jeżeli poleci timeout albo 401/403?. Zobacz w innych miejscach jak to john zrobił.

Reply:
poprawiłem

Reply:
tego nie ma przecierz, john zrobił wylogowanie przy errorze